nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/aggregates/TestAggregates.java

```diff
@@ -502,76 +248,7 @@ public void testAggregateDateHistogramQuery() throws Exception {
-                        "            \"adjust_pure_negative\" : true,\n" +
-                        "            \"boost\" : 1.0\n" +
-                        "          }\n" +
-                        "        }\n" +
-                        "      ],\n" +
-                        "      \"disable_coord\" : false,\n" +
-                        "      \"adjust_pure_negative\" : true,\n" +
-                        "      \"boost\" : 1.0\n" +
-                        "    }\n" +
-                        "  },\n" +
-                        "  \"_source\" : {\n" +
-                        "    \"includes\" : [\n" +
-                        "      \"_id\"\n" +
-                        "    ],\n" +
-                        "    \"excludes\" : [ ]\n" +
-                        "  },\n" +
-                        "  \"aggregations\" : {\n" +
-                        "    \"created_filter\" : {\n" +
-                        "      \"filter\" : {\n" +
-                        "        \"match_all\" : {\n" +
-                        "          \"boost\" : 1.0\n" +
-                        "        }\n" +
-                        "      },\n" +
-                        "      \"aggregations\" : {\n" +
-                        "        \"created\" : {\n" +
-                        "          \"date_histogram\" : {\n" +
-                        "            \"field\" : \"dc:created\",\n" +
-                        "            \"format\" : \"yyy-MM\",\n" +
-                        "            \"time_zone\" : \"UTC\",\n" +
-                        "            \"interval\" : \"month\",\n" +
-                        "            \"offset\" : 0,\n" +
-                        "            \"order\" : {\n" +
-                        "              \"_count\" : \"desc\"\n" +
-                        "            },\n" +
-                        "            \"keyed\" : false,\n" +
-                        "            \"min_doc_count\" : 0\n" +
-                        "          }\n" +
-                        "        }\n" +
-                        "      }\n" +
-                        "    }\n" +
-                        "  }\n" +
-                        "}", //
+        assertEqualsEvenUnderWindows("{\"from\":0,\"size\":10,\"query\":{\"match_all\":{\"boost\":1.0}},\"post_filter\":{\"bool\":{\"must\":[{\"bool\":{\"should\":[{\"range\":{\"dc:created\":{\"from\":1470009600000,\"to\":1472688000000,\"include_lower\":true,\"include_upper\":false,\"format\":\"epoch_millis\",\"boost\":1.0}}}],\"adjust_pure_negative\":true,\"boost\":1.0}}],\"adjust_pure_negative\":true,\"boost\":1.0}},\"_source\":{\"includes\":[\"_id\"],\"excludes\":[]},\"aggregations\":{\"created_filter\":{\"filter\":{\"match_all\":{\"boost\":1.0}},\"aggregations\":{\"created\":{\"date_histogram\":{\"field\":\"dc:created\",\"format\":\"yyy-MM\",\"time_zone\":\"UTC\",\"interval\":\"month\",\"offset\":0,\"order\":[{\"_count\":\"desc\"},{\"_key\":\"asc\"}],\"keyed\":false,\"min_doc_count\":0}}}}}}",
```

Comment:
Does it make sense to have this expected request as a file ? I could create a NXP.

Reply:
there are a lot of these asserts, it will be cleaner to have files but it will make  harder to view the request representation of a query and to update the representation which changes on each elasticsearch upgrade.